### ISSUE TITLE
[Quest][Northrend] High Commander Halford Wyrmbane quest

### ISSUE THREAD

**Issue by alderling** [dalaranwow/dalaran-wow#5317]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
In dragonblight, this part of a chain (quest id:12174) is meant to automatically fly you to Wintergarde Keep. However nothing happens upon accepting quest.
**Expected behaviour**: Tell us what should happen instead
Once you accept quest a unique taxi is supposed to fly you to wintergarde.
**Steps to reproduce the problem**:

1. Go to star's rest
2. accept quest from guy in tent
3. follow quest to flight master, accept and you will see issue

**Include proofs for this behaviour**
http://www.wowhead.com/quest=12174/high-commander-halford-wyrmbane#comments
You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**
quest id:12174
You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**
